vector/src/main/res/layout/item_poll_option.xml

```diff
@@ -36,34 +36,23 @@
         android:layout_marginStart="12dp"
         android:layout_marginTop="16dp"
         android:layout_marginEnd="12dp"
-        app:layout_constraintEnd_toEndOf="@id/optionWinnerImageView"
+        app:layout_constraintEnd_toStartOf="@id/optionVoteCountTextView"
         app:layout_constraintStart_toEndOf="@id/optionCheckImageView"
         app:layout_constraintTop_toTopOf="parent"
         tools:text="@sample/poll.json/data/answer" />
 
-    <ImageView
-        android:id="@+id/optionWinnerImageView"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="16dp"
-        android:layout_marginEnd="16dp"
-        android:contentDescription="@string/a11y_poll_winner_option"
-        android:src="@drawable/ic_poll_winner"
-        android:visibility="gone"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        tools:visibility="visible" />
-
     <TextView
         android:id="@+id/optionVoteCountTextView"
```

Comment:
Looking at [Figma](https://www.figma.com/file/tepA3ELWBE5v4O2WB8PqWn/Polls?node-id=1242%3A63420&t=O98eYEVP5YpGfwCy-0), we also need to update the color of the text for winning options.

Reply:
Oh, good catch!